pyramid_oereb/contrib/data_sources/standard/models/main.py

```diff
@@ -263,12 +272,14 @@ class GeneralInformation(Base):
         id (str): identifier and primary key, used in the database only
         title (dict): The title of the general information (multilingual)
         content (dict): The actual information (multilingual)
+        extract_index (int): index to sort the items in the extract
     """
     __table_args__ = {'schema': app_schema_name}
     __tablename__ = 'general_information'
     id = Column(String, primary_key=True)
     title = Column(JSONType, nullable=False)
     content = Column(JSONType, nullable=False)
+    extract_index = Column(Integer)
```

Comment:
Why is "extract_index" implemented differently than in class Disclaimer?

Reply:
At first I had used an auto-incremented extract_index if not provided (to use  a non-nullable field) but it seemed better to have an optional field.
I harmonized both definitions.